Google.Api.Generator.Utils/Roslyn/Keywords.cs

```diff
@@ -100,5 +100,7 @@ public static class Keywords
             );
 
         public static string PrependAtIfKeyword(string word) => s_keywords.Contains(word) ? $"@{word}" : word;
+
+        public static bool IsKeyword(string word) => s_keywords.Contains(word);
```

Comment:
nit: Use this method right above, on the PrependAtIfKeyword, and maybe do a quick search to see if it can be used somehwere else.

Reply:
Done.